1.segment_images/scripts/3.segmentation_decoupling.py

```diff
@@ -68,11 +95,19 @@
     reconstruction_dict_path = pathlib.Path(
         mask_path / "nuclei_reconstruction_dict.npy"
     ).resolve(strict=True)
+    distance_threshold = 20
 elif compartment == "cell":
     mask_file_path = pathlib.Path(mask_path / "cell_masks.tiff").resolve()
     reconstruction_dict_path = pathlib.Path(
         mask_path / "cell_reconstruction_dict.npy"
     ).resolve(strict=True)
+    distance_threshold = 50
+elif compartment == "organoid":
+    mask_file_path = pathlib.Path(mask_path / "organoid_masks.tiff").resolve()
+    reconstruction_dict_path = pathlib.Path(
+        mask_path / "organoid_reconstruction_dict.npy"
+    ).resolve(strict=True)
+    distance_threshold = 100
```

Comment:
What is distance threshold in the context of this PR? I originally assumed you were trying to relate objects together.